medicines/pars-upload/README.md

```diff
@@ -18,6 +18,35 @@ To understand how to use it, see the [example use of the form][example doc].
 
 [example doc]: ./docs/example.md
 
+
+## Running acceptance tests
+
+Browser based acceptance tests are run using [cypress](https://www.cypress.io).
+
+```sh
+yarn test-e2e
+```
+
+## Authentication
+
+The pages are protected by Azure Active Directory. Sign In will display a popup window where the normal Microsoft Identity Authentication (the same as office 365) will present itself.
+
+
+
+For development, this can be disabled by running yarn with the disable auth flag enabled.
+```sh
+yarn && NEXT_PUBLIC_DISABLE_AUTH=true yarn dev
+```
+or when running acceptance tests..
+```sh
+yarn && NEXT_PUBLIC_DISABLE_AUTH=true yarn test-e2e
+```
+
+Onward calls to upload the document, use JWT tokens produced by the authority configured in the environment variables in the .env file. Use the following command to get .env from Azure Key Vault.
```

Comment:
This doesn't read well because of the first comma.
I'm not sure _"onward"_ clarifies anything either – I'm not even sure what it means here.
If it's [#spon][spon] paid for by Disney, that's fine but we should declare it and I'd like my cut please.

I think it could be more precise about which environment variable defines the authority,
and I would `` `backtick` `` all the instances of `.env`.

Also, I split the sentences onto separate lines in Markdown as the Github renderer ignores single line breaks.
It also makes it easier to comment on specific lines
and it means future diffs can highlight more precisely within a paragraph too.

[spon]: https://www.urbandictionary.com/define.php?term=%23spon

Reply:
Actually, the "onward" calls aren't development yet, so I'm going to take that info out of this paragraph.
👍 re the separate lines but same paragraph